src/Main.java

```diff
@@ -39,37 +38,48 @@ public static void main(String[] args) {
         System.out.println();
 
         System.out.println("Задача 3");
+
 //          Три учительницы закупили все вместе 480 листов бумаги на все три класса.
 //          Посчитайте, сколько достанется листов каждому ученику.
 //          Результат задачи выведите в консоль в формате: «На каждого ученика рассчитано … листов бумаги».
         int pupilLP=23;    //  Людмилы Павловны — 23 ученика
         int pupilAS=27;    //  Анны Сергеевны — 27 учеников
         int pupilEA=30;    //  Екатерины Андреевны — 30 учеников
-        float allList=480;  //  закупили все вместе 480 листов бумаги
+        float allList=480f;  //  закупили все вместе 480 листов бумаги
         float list=allList/(pupilLP+pupilAS+pupilEA);
         System.out.println("На каждого ученика рассчитано "+list+" листов бумаги");
         System.out.println();
 
         System.out.println("Задача 4");
 //  Производительность машины для изготовления бутылок — 16 бутылок за 2 минуты.
-        float efficiency=16f;                    //Производительность машины для изготовления бутылок в 2 минуты
-        float efficiency1=efficiency/2*20;              //за 20 минут
-        float efficiency2=efficiency/2*60*24;           //за сутки
-        float efficiency3=efficiency/2*60*24*3;         //за 3 дня
-        float efficiency4=efficiency/2*60*24*30;        //за 1 месяц
-        System.out.println("За 20 минут машина произвела "+efficiency1+" штук бутылок ");
-        System.out.println("За сутки машина произвела "+efficiency2+",штук бутылок");
-        System.out.println("За 3 дня машина произвела "+efficiency3+",штук бутылок");
-        System.out.println("За 1 месяц машина произвела "+efficiency4+",штук бутылок");
+
+        float efficiencyPer2min=16f;
+        int minutesPerHour=60;
+        int hourPerDay=24;
+        int dayPerMouth=30;
+        float efficiency=efficiencyPer2min/2;                                           //Производительность машины для изготовления бутылок в минуту
+        float efficiencyPer20min=efficiency*20;                                         //за 20 минут
+        float efficiencyPerDay=efficiency*minutesPerHour*hourPerDay;                    //за сутки
+        float efficiencyPerTreeDay=efficiency*minutesPerHour*hourPerDay*3;              //за 3 дня
+        float efficiencyPerMouth=efficiency*minutesPerHour*hourPerDay*dayPerMouth;      //за 1 месяц
```

Comment:
```suggestion
        float efficiencyPerMouth=10*efficiencyPerTreeDay;
```
используем переменные, а не считаем каждый раз одно и то же